broker/message.py

```diff
@@ -35,6 +35,9 @@
 
 TYPE_LUT = { v:k for k,v in MESSAGE_TYPES.items() }
 
+class MQTTSNParseException(Exception):
+    pass
```

Comment:
Look where this exception gets handled at line 126. If this exception gets thrown, we log it.

https://github.com/jamesmunns/pymqttsn-broker/blob/mild-refactor/broker/message.py#L125-L127